robot/basestation/templates/elements/feed/Navbar.html

```diff
@@ -45,33 +45,32 @@
       Theme
     </button>
     <div class="dropdown-menu">
-      <a id="theme-red" class="dropdown-item">Red</a>
-      <a id="theme-green" class="dropdown-item">Green</a>
-      <a id="theme-blue" class="dropdown-item">Blue</a>
+      <a id="theme-mantis" class="dropdown-item">mantis</a>
+      <a id="theme-lofi" class="dropdown-item">lofi</a>
     </div>
   </div>
 
   <div class="col-md-1">
-    <div class="row border border-white" style="color:white">
+    <div class="row border border-white" style="color:white;">
```

Comment:
We should remove the inline css and make it it's own class. Something that would make semantic sense like `pds-stats`?
Also the spacing of these attributes and values is inconsistent, in general it should be formatted like `attribute="value"`
Since this is enforcing coding standards if I see an instance of something I have already explicity mentioned in a comment I will just comment "CS" to better keep track of all instances

Reply:
Fixed